_posts/2022-01-06-jbossws-log4j-cve.md

```diff
@@ -0,0 +1,32 @@
+---
+layout:     post
+title:       "Log4j CVE and JBossWS project"
+subtitle:   ""
+date:       Jan 6,2022
+author:     JBossWS Team
+---
+After [log4j cve issues](https://logging.apache.org/log4j/2.x/security.html) are reported, we carefully
+reviewed the usage in JBossWS projects and found we only use log4j dependencies in command line tools, client side
+and testsuite. These are places we use log4j dependency:
+- [jbossws-common-tools](https://github.com/jbossws/jbossws-common-tools)
+
+  It is used to log debug or error message when run wsconsume or wsprovide command line tool. 
+  Because this tool will be finally used in command line and out of JavaEE/Jakarta EE container 
+  server without touching anything about JNDI and JMS, so these CVE issues won't impact jbossws project. 
+  But in some edge or some unluck case and if other project has a dependency or transitive dependency to org.jboss.ws:jbossws-common-tools or run this
+  client tool in server side, we already upgraded log4j version to the latest 2.17.1.
```

Comment:
I suggest replacing this paragraph with the following.

It is used to log debug and error messages when running the wsconsume or wsprovide command line tools. 
Because these tools only run on the command line and do not run in the JavaEE/Jakarta EE container 
server, this CVE issue doesn't impact the jbossws projects, however
we have upgraded org.jboss.ws:jbossws-common-tools to use log4j version 2.17.1.

Reply:
@rsearls  Thanks for review !